recipes/treeinterpreter/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "treeinterpreter" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 86266c43a82acc79ae7bf1bac5f8c30990e6492487a9727811346dd1f62aa074
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```

Comment:
It looks like this package has unlisted dependencies on `numpy` and `sklearn`:

https://github.com/andosa/treeinterpreter/blob/master/treeinterpreter/treeinterpreter.py#L2-L3

these don't trip the import test since the module import doesn't really run anything.

Reply:
added numpy and scikit-learn 0.17+ (based on README)